server/views/pages/goal/update/review.njk

```diff
@@ -113,7 +113,7 @@
         <div class="govuk-form-group govuk-!-margin-bottom-4">
           {{ govukButton({
             type: "a",
-            href: "/plan/" + prisonerSummary.prisonNumber + "/goals/" + data.reference + "/update",
+            href: "/plan/" + prisonerSummary.prisonNumber + "/goals/" + data.goalReference + "/update",
```

Comment:
The integration test was failing although I don't know why it would suddenly start to fail now? When I made this change it passed. It seems like the reference wasn't being passed, see urls before and after the change.
![Screenshot 2024-10-04 at 10 33 56](https://github.com/user-attachments/assets/783cb702-ee12-4773-a36e-5992fbd61885)
![Screenshot 2024-10-04 at 10 35 04](https://github.com/user-attachments/assets/086789e2-0fab-4f73-9bd5-a845ba12e613)

Reply:
Thats weird! The property in the nunjucks is definitely wrong, and you're doing the right thing in fixing it 👍 
but I dont know why an integration test should start to fail ... 🤔 🤷‍♂️ 
Oh well, this is a good fix, so well spotted 👍